openlibrary/plugins/openlibrary/js/index.js

```diff
@@ -173,12 +177,12 @@ jQuery(function () {
     }
     // conditionally loads Goodreads import based on class in the page
     if (document.getElementsByClassName('import-table').length) {
-        import('./goodreads_import.js')
+        import(/* webpackChunkName: "goodreads" */'./goodreads_import.js')
             .then(module => module.initGoodreadsImport());
     }
     // conditionally loads Related Carousels based on class in the page
     if (document.getElementsByClassName('RelatedWorksCarousel').length) {
-        import('./carousels_partials.js')
+        import(/* webpackChunkName: "carousel-partials" */'./carousels_partials.js')
```

Comment:
```suggestion
        import(/* webpackChunkName: "carousels-partials" */'./carousels_partials.js')
```

(Unrelated: This file is poorly named).